go/private/mode.bzl

```diff
@@ -113,6 +114,7 @@ def get_mode(ctx, go_toolchain, cgo_context_info, go_config_info):
         goos = goos,
         goarch = goarch,
         tags = tags,
+        package_conflict_is_error = package_conflict_is_error,
```

Comment:
This flag should be removed in `v0.25`, so I'd rather hide it a little better. Let's move it to the `go_context` object with the name `_package_conflict_is_error`, in the `# Private` section below `_ctx`.

Reply:
Done.